src/main/java/com/google/step/servlets/HomePageServlet.java

```diff
@@ -58,6 +63,10 @@
 
   private static final String VOTE_SORT = "votes";
   private static final String NEW_SORT = "new";
+  private static final String DISTANCE_SORT = "distance";
+
+  private String latitude = null;
+  private String longitude = null;
```

Comment:
why are these 2 strings private global variables? If 2 calls are made to the servlet at the same time, they will share the variable